.pre-commit-config.yaml

```
@@ -0,0 +1,11 @@
+repos:
+  - repo: https://github.com/doublify/pre-commit-rust
+    rev: v1.0 # this specifies the version
+    hooks:
+     - id: fmt
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)